DopplerJobsServer/appsettings.development.json

```diff
@@ -44,13 +44,13 @@
   },
   "DopplerCurrencyServiceSettings": {
     "Url": "https://apis.fromdoppler.com/currency/conversion/",
-    "CurrencyCodeList": [ "ARS" ],
+    "CurrencyCodeList": [ "ARS", "MXN", "COP" ],
     "InsertCurrencyQuery": "[dbo].[InsertNewCurrencyRate]",
     "HolidayRetryCountLimit": 5
   },
   "DopplerSapConfiguration": {
-    "CurrencyEndpoint": "http://localhost:5002/sap/billing/SetCurrencyRate",
-    "BillingEndpoint": "http://localhost:5002/Billing/CreateBillingRequest"
+    "CurrencyEndpoint": "http://localhost:5002/billing/SetCurrencyRate",
+    "BillingEndpoint": "http://localhost:5002/billing/CreateBillingRequest"
```

Comment:
why is this?

Reply:
you are right about the question It doesn't make sense, I revert this change